bin/tools.py

```diff
@@ -505,28 +496,7 @@ def build_parser():
     genparser.add_argument(
         '--timeout', '-t', type=int, help='Override the default timeout. Default: 30.'
     )
-    genparser.add_argument(
```

Comment:
Hmmm, I do like not accidentally overwriting samples, for example with more/less precise floating point numbers.

On the other hand, we assume everything is in git anyway so any diffs will clearly show up and hard to miss. Removing this probably ends up simpler, more predictable, and anyway safe enough.